experiments/model_test.txt

```diff
@@ -0,0 +1,43 @@
+Experiment file for model: test 
```

Comment:
kinda doesn't matter for this but generally you want check program outputs into version control like this. They can get very big over time (especially the models) and git isn't really built for storing large amounts of data.

Reply:
You can exclude them from git by adding htem to `.gitignore`